Code Samples/Fib/.vscode/tasks.json

```diff
@@ -4,46 +4,52 @@
     "version": "2.0.0",
     "tasks": [
         {
-            "label": "build",
+            "label": "build (Unix)",
             "type": "shell",
+            "command": "make",
             "group": {
```

Comment:
This is still defaulting to the `build (Unix)` even if it's Windows. Was there a problem with the previous per-OS task.json?